acme.go

```diff
@@ -66,6 +67,13 @@ const (
 	boulderBuildIDPath    = "/build"
 )
 
+var (
+	// ErrUnsupportedRSABitLen reports whether an unrecognized RSA key size was used
+	ErrUnsupportedRSABitLen = errors.New("unsupported RSA bit length")
```

Comment:
The errors are anyways being sent up the stack to the users. Do you prefer for them using string comparisons in case they want to handle any of the errors?